flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/JdbcSink.java

```diff
@@ -107,5 +108,9 @@ public static <T> SinkFunction<T> exactlyOnceSink(
                 exactlyOnceOptions);
     }
 
+    public static <IN> JdbcSinkBuilder<IN> newSink() {
```

Comment:
how about 
```
public static <IN> JdbcSinkBuilder<IN> newSinkBuilder() {

```

Reply:
changing it to builder.. all other methods should be deprecated after